src/io.c

```diff
@@ -59,9 +58,13 @@ flush_event_buffer (gpointer data);
 static void
 free_cmd_req (gpointer data);
```

Comment:
The struct is still around just a bit smaller. It's used as the task data (g_task_set_task_data)